### ISSUE TITLE
Horizontal scaling?

### ISSUE THREAD

**Issue by cdhanna** [rajnandan1/kener#43]
**Is your feature request related to a problem? Please describe.**
I am worried that if I adopt this product, it won't scale well in high traffic. I would want to be able to have multiple node servers handling requests in a round-robin fashion, hosted on a fleet of containers. But from what I gather from looking at the docs and code, this isn't possible. If I run multiple instances, I think I'd be running multiple _collectors_, which would duplicate metrics.

**Describe the solution you'd like**
I would like to be able run the server in "view only mode", and disable metric collection. This way, I can run _one_ metric collector, and leave the rest of the instances as view only, so that I can scale out new instances to meet traffic needs. 

**Describe alternatives you've considered**
1. modify the source code myself, 
2. simply have 1 instance and hope it handles traffic load,
3. simply have many instances, and don't care that the metrics are duplicated. This would mean that different users may have a slightly different metric reporting experience. 

**Additional context**
I think this project is awesome! I would love to adopt it for my use case :)

**Comment by rajnandan1** [comment]
Yes, I have working on separating the collection and display part in local. 

Taking longer than usual to raise the PR

Will close the issue once raised and merged.

**Comment by rajnandan1** [comment]
Hi about to 1 year since I have said I will work on this, but with the latest release it is doable. Full honesty I have not tried it my self

You can specify postgres database

You can run `node src/lib/server/startup.js` in one of the servers. This will collect metrics

You can host just run the frontend using `node main.js`

Please check the docs https://kener.ing/docs/deployment/#npm and tell me if it worked

**Comment by rajnandan1** [comment]
This is now possible
